### PR TITLE
Copying proposal if the existing proposal already has a session

### PR DESCRIPTION
**Pull request by iambibhas**

### CODE REVIEW

File: funnel/views/proposal.py [hunk 1]
```diff
@@ -390,7 +390,10 @@ def proposal_moveto(profile, space, proposal):
     if proposal_move_form.validate_on_submit():
         target_space = proposal_move_form.target.data
         if target_space != proposal.proposal_space:
-            proposal.current_access().move_to(target_space)
+            if proposal.state.MOVABLE:
+                proposal.current_access().move_to(target_space)
+            else:
+                proposal = proposal.current_access().copy_to(target_space)
```
Comment (jace) [inline funnel/views/proposal.py]: This has to be in the UI as well: don't show "Move to" if you can only make a copy.

**Review by jace — CHANGES_REQUESTED** [review]

**Comment by jace** [comment]
@iambibhas When you make a copy, "user" must be the person who made the copy. "speaker" can remain as at present. We have to record the person who requested the copy operation.

**Comment by jace** [comment]
Can you clean this up for merger?

**Comment by iambibhas** [comment]
Pushed some fixes. But UI has some serious issue. /cc @vidya-ram 
funnel-issue

**Comment by jace** [comment]
What should we do with this?

**Comment by jace** [comment]
As per #327, proposals are no longer attached to projects. There is a separate `ProposalSubmission` instead.

**Comment by jace** [comment]
@iambibhas this is the issue that broke for Zainab last week. Let's close this?

**Review by jace — CHANGES_REQUESTED** [review]
This has a few issues.

**Comment by jace** [comment]
This is now severely out of date, so we'll have to redo this as a new PR.

**Pull request closed without merging**